core/src/main/java/tachyon/master/MasterInfo.java

```diff
@@ -823,6 +833,7 @@ public boolean _rename(int fileId, TachyonURI dstPath, long opTimeMs)
       srcInode.setName(dstComponents[dstComponents.length - 1]);
       ((InodeFolder) dstParentInode).addChild(srcInode);
       dstParentInode.setLastModificationTimeMs(opTimeMs);
+      mMasterSource.incFilesRenamed();
```

Comment:
Should we also have a `renameOps` counter? If the `ops` counters are counting successful method calls, it should be different from the total number of files renamed due to returning success if the two paths are already equal.